guide/getting-started/technology-primer.md

```diff
@@ -73,264 +79,265 @@ It's both. Lowercase bitcoin refers to the currency. So you may have 1 bitcoin i
-
-Wallet software typically hides private keys from users, as there is no need to interact with them directly.
+Instead of processing each transaction individually, the Bitcoin network bundles them into [blocks]({{'/guide/glossary/#hash' | relative_url }}). A new block is created roughly every 10 minutes and can only contain a certain amount of transactions due to a strict file size limit, so transactions with the highest fees typically get chosen first. The new block must link to the one before it. Since this is a continuous process, a chain of blocks forms, hence the name blockchain.
 
+Once a transaction is in a block that the network has accepted, it is considered to have one confirmation. With every new block created, the number of confirmations increases. After 6 confirmations, we consider a transaction fully settled.
 </div>
 
-## How are private keys and addresses connected?
+## How is the blockchain secured?
 
 <div class="center" markdown="1">
 
 {% include image.html
-   image = "/assets/images/guide/getting-started/technology/encryption.jpg"
-   retina = "/assets/images/guide/getting-started/technology/encryption@2x.jpg"
-   alt-text = "Chart showing encryption and decryption of a message during communication"
+   image = "/assets/images/guide/getting-started/technology/mining-a-block.jpg"
+   retina = "/assets/images/guide/getting-started/technology/mining-a-block@2x.jpg"
+   alt-text = "Transactions moving from the mempool into a block"
    width = 400
    height = 400
    layout = "float-right-desktop"
 %}
 
-A private key is used to scramble information. The matching public key (which we call an address in Bitcoin) can unscramble the information again. This allows for the transfer of information without any third party being able to understand it.
+While miners check that the transactions included in the block are valid, they must also be kept accountable that the information they are submitting is correct. While checking transactions for a new block, miners must also race to solve a cryptographic puzzle as proof that they have correctly done the verification. This process is called Proof of Work, and it is fundamental to the security of the blockchain. As compensation, the first miner to submit a valid block to the network, solving the puzzle, will earn all the fees from the transactions in the block and a block reward.
 
-Note that there is a strong element of identity in this mechanism, as it ensures that only the recipient can unscramble the message while also verifying that the message came from the expected sender. This requires that both parties securely exchange public keys with each other in a secure way.
+We also call the Bitcoin blockchain "base layer" due to the strong security it provides.
```

Comment:
A bit of a floating sentence. Not sure how strong security leads to the name "base layer".